lib/steps.js

```diff
@@ -217,7 +214,7 @@ When(/^the user gets to ?(?:the)?(?: ?question)?(?: id)? "([^"]+)" with this dat
         try {
           let { question_has_id, id, id_matches } = await scope.examinePageID( scope );
           // Some errors happen in there to be closer to the relevant code. We'd have to try/catch anyway.
-          let leftovers = await scope.processVar(scope, { var_data: remaining_vars, id });
+          let leftovers = await scope.processVar(scope, { var_data, id });
```

Comment:
`processVar` is returning errors now, not remaining variables. I think that changes how this part of the code works.

Reply:
You're right, `leftovers` isn't really useful anymore. Done!